WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/StartingInfo/PromptProgressBar/ProductCreationAIBarProgressStyle.swift

```diff
@@ -0,0 +1,38 @@
+import SwiftUI
+
+struct ProductCreationAIBarProgressStyle: ProgressViewStyle {
+
+    var color: Color = .gray
+
+    func makeBody(configuration: Configuration) -> some View {
+
+        let progress = configuration.fractionCompleted ?? 0.0
+
+        GeometryReader { geometry in
+            VStack(alignment: .leading) {
+                RoundedRectangle(cornerRadius: Layout.radius)
+                    .fill(color.opacity(0.5))
```

Comment:
Super nit: This can be moved to an `Constants` enum.

Reply:
e301da010ad2a7f7a7ab45dede4e178478901b2c